nautobot/ipam/models.py

```diff
@@ -679,19 +676,23 @@ def get_first_available_ip(self):
     def get_utilization(self):
         """Get the child prefix size and parent size.
 
-        For Prefixes with a status of "container", get the number child prefixes. For all others, count child IP addresses.
+        For Prefixes with a type of "container", get the number child prefixes. For all others, count child IP addresses.
```

Comment:
IMHO as part of the 2.0 changes we should think about splitting this function into a `get_prefix_utilization` and `get_ip_address_utilitization` function.